InvenTree/order/models.py

```diff
@@ -931,12 +945,6 @@ class Meta:
         help_text=_('Unit price'),
     )
 
-    def price_converted(self):
```

Comment:
Question: Are you sure we do not need this for pricing?

Reply:
I could not find a reference to this function anywhere

Reply:
We will find out shortly if not ;-) fully covered, API driven is the future.